release_package.yml

```diff
@@ -23,9 +24,15 @@
 
     - set_fact:
         copr_package_info: "{{ copr_package_info.stdout | from_json }}"
+      when: "'No package with name' not in copr_package_info.stderr"
 
     - set_fact:
         copr_package_version: "{{ copr_package_info['latest_build']['pkg_version'] }}"
```

Comment:
Would ```"{{ copr_package_info['latest_build']['pkg_version'] | default("") }}"``` work?

Reply:
It will still try to access the first part of the dictionary, which in this case is a string and fail out.